Source/Core/DolphinLib.vcxproj

```diff
@@ -27,6 +27,12 @@
       <Project>{41279555-f94f-4ebc-99de-af863c10c5c4}</Project>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup>
+    <ClCompile Include="VideoCommon\VertexLoaderARM64.cpp" />
```

Comment:
I think this should go in the props file?

Reply:
That shouldn't have been commited at all; I added it because VS didn't want to format the code unless it was included in the solution, and right-clicking it to add it to the solution puts it there...